library/multichoicerecyclerview/src/main/java/com/davidecirillo/multichoicerecyclerview/MultiChoiceToolbarHelper.java

```diff
@@ -31,13 +33,13 @@
         }
     }
 
-    void updateToolbar(int itemNumber) throws IllegalStateException {
-        if (itemNumber == ZERO) {
+    void updateToolbar(int selectedItemCount) throws IllegalStateException {
```

Comment:
The variable-name wasn't clear to be so I changed it. I think this is clearer now.

I can revert it, if you like. Should I?